### DISCUSSION TITLE
how to Sort the size of response packets?

### DISCUSSION THREAD

**Discussion by 2077himasi** [projectdiscovery/httpx#1309] [Q&A]
like 
httpx -l 1.txt -path /v1/api  -csv -o 2.csv
I hope the results of CSV are sorted according to the size of response packets?. What should I do?

**Comment by MetzinAround** [comment]
Are you able to sort them after the file is created? With a CSV you could sort them in excel or sheets.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
